contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpScanBatchCreator.java

```diff
@@ -161,8 +161,8 @@ public void bind(ManagedScanFramework framework) { }
         }
       } else if (paginatorConfig != null) {
         /*
-        * If the paginator is not null and generated a list of URLs, we create
-        * a new batch reader for each URL.  In the future, this could be parallelized in
+        * If the paginator is not null we create a new batch reader for each
+        * URL that it generates.  In the future, this could be parallelized in
```

Comment:
```suggestion
        * URL that it generates. In the future, this could be parallelized in
```

Reply:
@vdiravka I didn't realise that two spaces after a period has gone out of fashion, but it seems it started dying shortly after the typewriter :)